snapshot-tests/src/test/java/io/eventuate/examples/tram/ordersandcustomers/snapshottests/SnapshotTestConfiguration.java

```diff
@@ -0,0 +1,30 @@
+package io.eventuate.examples.tram.ordersandcustomers.snapshottests;
+
+import org.springframework.boot.autoconfigure.http.HttpMessageConverters;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.http.converter.HttpMessageConverter;
+import org.springframework.http.converter.json.MappingJackson2HttpMessageConverter;
+import org.springframework.web.client.RestTemplate;
+
+import java.util.Arrays;
+import java.util.List;
+
+@Configuration
+public class SnapshotTestConfiguration {
+
+  @Bean
+  public RestTemplate restTemplate(HttpMessageConverters converters) {
```

Comment:
Is this needed?

Reply:
RestTemplate is needed, other things I removed.